lib/TimeSeries/AdaptiveFilter.pm

```diff
@@ -0,0 +1,210 @@
+package TimeSeries::AdaptiveFilter;
+
+=head1 NAME
+
+TimeSeries::AdaptiveFilter - Adaptive filter for data stream with possible outliers
+
+=cut
+
+our $VERSION = '0.02';
+
+=head1 VERSION
+
+Version 0.02
+
+=head1 STATUS
+
+=begin HTML
+
+<p>
+    <a href="https://travis-ci.org/binary-com/perl-TimeSeries-AdaptiveFilter"><img src="https://travis-ci.org/binary-com/perl-TimeSeries-AdaptiveFilter.svg" /></a>
+</p>
+
+=end HTML
+
+=cut
+
+use strict;
+use warnings;
+
+use List::Util qw(sum max);
+
+use parent qw/Exporter/;
```

Comment:
Do you need to inherit from `Exporter` or do you just need an `import` method? In the latter case better to

```perl
use Exporter qw/import/;
```